src/services/lnd/utils-ln-service.ts

```diff
@@ -0,0 +1,44 @@
+import lnService from "ln-service"
+
+import { getLnds } from "./utils"
+
+const lndsParamsAuth = getLnds({ type: "offchain", active: true })
+
+const lnd1 = lndsParamsAuth[0].lnd
+const lnd2 = lndsParamsAuth[1].lnd
+
+async function getPendingHtlcCountLnd1(): Promise<{ [channelId: string]: number }> {
+  const channels = await lnService.getChannels({ lnd1 })
+  const pendingHtlcCounts: { [channelId: string]: number } = {}
+  channels.forEach((channel: { id: string; pending_payments: any[] }) => {
+    pendingHtlcCounts[channel.id] = channel.pending_payments.length
+  })
+  return pendingHtlcCounts
+}
+
+async function getPendingHtlcCountLnd2(): Promise<{ [channelId: string]: number }> {
+  const channels = await lnService.getChannels({ lnd2 })
+  const pendingHtlcCounts: { [channelId: string]: number } = {}
+  channels.forEach((channel: { id: string; pending_payments: any[] }) => {
```

Comment:
here probably you can use a reducer instead the use of 2 methods (unless you need the pending htlcs by channel id), something like:

```
return channels.reduce((total, channel) => {
    return total + channel.pending_payments.length;
}, 0)
```

Reply:
`getTotalPendingHtlcCount` is one function now